app/transformers/api/current/patient_transformer.rb

```diff
@@ -28,7 +34,10 @@ def to_nested_response(patient)
             Api::Current::Transformer.to_response(phone_number).except('patient_id')
           end,
           'business_identifiers' => patient.business_identifiers.map do |business_identifier|
-            Api::Current::Transformer.to_response(business_identifier).except('patient_id')
+            Api::Current::Transformer
+              .to_response(business_identifier)
+              .except('patient_id')
+              .merge('metadata' => business_identifier.metadata&.to_json)
```

Comment:
Let's avoid the usage of `&.` 

If metadata is not present., we should not add the key to the response.

Reply:
I believe the key will already be present with a `nil` value

Reply:
refactored, added a `PatientBusinessIdentifierTransformer`